src/redux/reducers/parentReducer.js

```diff
@@ -6,64 +6,146 @@ import {
 } from '../actions/parentActions';
```

Comment:
Not sure why this file was included in the PR when I didn't even add it

Reply:
Yeah, that's strange... everything looks good though. Thanks for your hard work, greatly appreciated!